src/frontend/src/pages/DesignReviewSummaryModal.tsx

```
@@ -0,0 +1,186 @@
+                          <DesignReviewMemberPill user={member} />
+                        </Grid>
+                      ))}
+                    </Grid>
+                  </Grid>
+                </Grid>
+              </Grid>
+            </Grid>
+          </Grid>
+        </Grid>
+
+        <Grid item xs={12}>
+          <FormControlLabel
+            label="Mark Design Review as Complete"
+            sx={{ marginBottom: 5 }}
+            control={
+              <Checkbox
+                checked={checked}
+                onChange={handleChange}
+                sx={{
+                  color: 'white',
+                  '&.Mui-checked': { color: 'white' }
+                }}
+              />
+            }
+          />
+        </Grid>
+
+        <Grid item xs={12}>
+          <Box
+            sx={{
+              position: 'absolute',
+              bottom: 10,
+              right: 10,
+              display: 'flex',
+              flexDirection: 'row'
+            }}
+          >
+            <Button
+              sx={{
+                color: '#474849',
+                backgroundColor: '#d9d9d9',
+                ':hover': { backgroundColor: '#A4A4A4' },
+                fontSize: 10,
+                marginLeft: 1,
+                fontWeight: 'bold'
+              }}
+              disabled={designReview.status !== DesignReviewStatus.DONE || !checked}
+            >
+              Schedule Another DR
+            </Button>
+            <Button
+              sx={{
+                color: '#474849',
+                backgroundColor: '#82e94b',
+                ':hover': { backgroundColor: '#559334' },
+                fontSize: 10,
+                marginLeft: 1,
+                fontWeight: 'bold'
+              }}
+              onClick={() => setShowStageGateModal(true)}
+              disabled={designReview.status !== DesignReviewStatus.DONE || !checked}
+            >
+              Stage Gate
+            </Button>
+            <Button
```

you should be using the NERSuccess and Fail Buttons here instead of hardcoding the styling. If you want you can add a prop to the NERButton prop to give it a white variant